ppafm/basUtils.py

```diff
@@ -404,6 +404,31 @@ def parseLvecASE(comment):
         lvec = None
     return lvec
 
+def create_at_array(atoms,lvec):
```

Comment:
I don't think this function is needed. Its functionality can be replaced by one line:
```python
atoms_array = np.append(np.array(atoms), np.array(lvec))
```

Reply:
I do not think so, as atoms is list of lists [e, x, y, z] or [e, x, y, z, q], where e is interger, and lvec is 4x3 matrix, where I on purpose want to get rid of the first row.